api/routes/roles/get.endpoint.js

```diff
@@ -16,11 +16,31 @@ describe('Roles endpoint', () => {
     unauthenticatedTest('get', '/roles');
     unauthorizedTest('get', '/roles');
 
+    it('returns expected number of roles given a system admin', async () => {
+      const authedClient = login('all-permissions');
+      const response = await authedClient.get('/roles');
+      expect(response.status).toEqual(200);
+      expect(response.data.length).toBe(6);
```

Comment:
I think this might be best as a snapshot

Reply:
Yep, this is a good case for snapshots.